scvi/distributions/_beta_binomial.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+
+import torch
+from pyro.distributions import BetaBinomial as BetaBinomialDistribution
+from torch.distributions import constraints
+from torch.distributions.utils import broadcast_all
+
+from ._constraints import open_interval, optional_constraint
+
+
+class BetaBinomial(BetaBinomialDistribution):
+    r"""Beta binomial distribution.
```

Comment:
```suggestion
    r"""``BETA`` Beta binomial distribution.
```
Let's tag this as beta for now.